Apps/Admin/Client/Program.cs

```diff
@@ -82,7 +98,8 @@ public static async Task Main(string[] args)
 
             builder.Services.AddBlazoredLocalStorage();
 
-            await builder.Build().RunAsync().ConfigureAwait(true);
```

Comment:
What's the point of this change?

Reply:
So that we can get the query string parameter and set the kc_idp_hint and go to the desired login type (IDIR or PHSA).